.golangci.yml

```diff
@@ -105,6 +105,7 @@ issues:
     - path: _test\.go
       linters:
         - gomnd
+        - funlen
```

Comment:
👍 good idea ! I thought it was enabled by default: it means that a lot of functions might starte to trigger alarms :)

Reply:
I had a lint error on a test function now too long

Reply:
And since it's a test it didn't seem judicious to remove test cases or to split this function, and I don't see the harm of disabling this rule for tests (in fact it's one of the exception cases given as example on the lint documentation)

Reply:
OK thabks for the explanation! Remind me something: might be related to the amount of managed cases for this pugin resource.
Looks good, what you said make perfect sense!